boards/FSM/Src/Io/Io_FlowMeters.c

```diff
@@ -0,0 +1,82 @@
+#include "main.h"
+#include "Io_SharedFreqOnlyPwmInput.h"
+#include "Io_FlowMeters.h"
+#include <assert.h>
+
+struct FlowMeters_Context
+{
+    struct FreqOnlyPwmInput *pwm_input;
+    TIM_HandleTypeDef *      htim;
+    HAL_TIM_ActiveChannel    active_channel;
+};
+
+static struct FlowMeters_Context primary_flow_meter, secondary_flow_meter;
+
+void Io_FlowMeters_Init(TIM_HandleTypeDef *htim)
+{
+    assert(htim != NULL);
+
+    primary_flow_meter.htim   = htim;
+    secondary_flow_meter.htim = htim;
+
+    primary_flow_meter.active_channel   = HAL_TIM_ACTIVE_CHANNEL_1;
```

Comment:
What is the difference between `HAL_TIM_ACTIVE_CHANNEL_1` and `TIM_CHANNEL_1`?

Reply:
https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/564#discussion_r411058703